contracts/ENSGovernor.sol

```diff
@@ -46,7 +47,7 @@ contract ENSGovernor is Governor, GovernorProposalThreshold, GovernorCountingSim
     function getVotes(address account, uint256 blockNumber)
         public
         view
-        override(IGovernor, GovernorVotes)
+        override(IGovernor, Governor)
```

Comment:
What's the motivation behind the changes to this file?

Reply:
Compiler complains;
```
Function needs to specify overridden contract "Governor".
Invalid contract specified in override list: "GovernorVotes".
```